packages/shared/src/utils/config/config.js

```diff
@@ -91,3 +91,27 @@ export const getSocketURL = () => {
     }
     return server_url;
 };
+
+export const checkAndSetStagingEndpoint = () => {
```

Comment:
Since it's no longer just staging, can we set this to `checkAndSetEndpointFromUrl`? It'd be more fitting on where it checks from too.